dev-python/msgpack/metadata.xml

```diff
@@ -9,4 +9,10 @@
     <remote-id type="pypi">msgpack-python</remote-id>
     <remote-id type="github">msgpack/msgpack-python</remote-id>
   </upstream>
+  <use>
+    <flag name="native-extensions">
+        Compiles native "C" extensions (speedups, instead of using python
+        backup code).
```

Comment:
The common term is 'fallback' (instead of 'backup'), I think.

Reply:
Yeah, that's true. I just copied this from the portage metadata.xml, I'll change that.